### DISCUSSION TITLE
REST API Orders question

### DISCUSSION THREAD

**Discussion by rkbjunior** [woocommerce/woocommerce#52083] [General]
Hello, I'm hoping someone can help me understand what is going on with my integration. Our accounting software uses the woocommerce rest api to send orders entered into our system up to woo. We just started discounting line items on our orders in our system. I've been sending them to woo like this:

`"line_items": [ {"name":"ProductName","product_id":"65","variation_id":"0","quantity":1,"subtotal":"699","total":"679" } ]`

This is an item for 699 regular with a 20 dollar discount applied to it directly.

However the response from WooCommerce is this:

`"line_items": [ { "id": 604989, "name": "ProductName", "product_id": 65, "variation_id": 0, "quantity": 1, "subtotal": "679.00", "total": "679.00", "sku": "ABC", "price": 679 } ]`

Why is it lowering the subtotal (price before discounts)? This doesn't make sense to me. Is this the default behavior? Version 9.3.1 legacy order system.

**Comment by barryhughes** [comment]
Sorry to hear you've hit some stumbling blocks, @rkbjunior. I'd recommend forming this into a [bug report](https://github.com/woocommerce/woocommerce/issues/new?template=1-bug-report.yml): that way, one of the team can properly assess, confirm if this is indeed a bug (or if there is actually some reason behind it), and prioritize accordingly.

**Reply by rkbjunior** [reply]
Thanks for the reply. I was actually able to identify a plugin that was causing this behavior.  My post here was mainly to confirm my understanding of how the subtotal field was supposed to work. The plugin in question is Avalara Tax for woocommerce. It is modifying the subtotal field after tax calculations. I've notified their support team of the issue.

**Reply by barryhughes** [reply]
Perfect, and thank you for following up with that note 👍🏼
